vumi/tests/helpers.py

```diff
@@ -1454,7 +1454,12 @@ def cleanup(self):
                 unclosed_managers.append(manager)
             if purge:
                 try:
-                    yield self._purge_riak(manager)
+                    # Create a new client for cleanup
+                    purge_manager = self.get_riak_manager(config={
+                        'bucket_prefix': manager.bucket_prefix
+                    })
```

Comment:
This is the part I'm not too certain about. Is there anything other than the bucket prefix that we need to worry about for cleaning up for tests?